es6-async-await/await.js

```diff
@@ -0,0 +1,56 @@
+import fetch from './fetch.js';
+
+// Keep track of app load time so each log message can be timed.
+// Log messages should all be approximately 1 second apart.
+const startTime = Date.now();
+const elapsed = () => `${Math.round((Date.now() - startTime) / 1000)}s -`;
+
+async function fetchOnce() {
+  try {
+    const response = await fetch('foo/bar.html');
+    const msg = await response.toString();
+    console.log(elapsed(), 'fetchOnce:', msg);
+  } catch (error) {
+    console.error(error);
+  }
+}
+
+async function fetchSeveral() {
+  try {
+    const response = await fetch('foo1/bar.html');
+    const msg = await response.toString();
+    console.log(elapsed(), 'fetchSeveral1:', msg);
+
+    const response2 = await fetch('foo1/bar.html');
```

Comment:
This isn't quite right. You are passing the wrong URL. Take a look at the original code.